pkg/kubelet/apis/cri/v1alpha1/runtime/api.pb.go

```diff
@@ -3183,6 +3183,9 @@ func (m *UInt64Value) GetValue() uint64 {
 type StorageIdentifier struct {
 	// UUID of the device.
 	Uuid string `protobuf:"bytes,1,opt,name=uuid,proto3" json:"uuid,omitempty"`
```

Comment:
somewhat related: do we plan to keep the UUID field?  If so, what is it used for now that we are passing the mount point?

Reply:
read the PR description, ignore :)